packages/cursorless-vscode/package.json

```diff
@@ -961,7 +961,7 @@
             ]
           }
         },
-        "cursorless.experimental.keyboard.modal.keybindings.shapes": {
+        "cursorless.experimental.keyboard.modal.keybindings.shape": {
```

Comment:
WDYT about changing:

> cursorless.experimental.keyboard.modal.keybindings.*

to

> cursorless.keyboard.bind.*

I understand every single word and why it is there in the original, but it feels like we could get away with something much more concise.

Reply:
I know we're leaning away from the "experimental" prefix, but it makes me feel a bit more comfortable with things like the gruff deprecation from plural to singular. If it weren't tagged experimental, I'd prob invest in auto-migration. Maybe we switch to that once we have a subset we're more confident in?

Reply:
Works for me.